lua/lazyvim/plugins/extras/lang/c.lua

```diff
@@ -0,0 +1,97 @@
+return {
+
+  -- Add C/C++ to treesitter
+  {
+    "nvim-treesitter/nvim-treesitter",
+    opts = function(_, opts)
+      if type(opts.ensure_installed) == "table" then
+        vim.list_extend(opts.ensure_installed, { "c", "cpp" })
+      end
+    end,
+  },
+
+  -- Correctly setup lspconfig for clangd 🚀
+  {
+    "neovim/nvim-lspconfig",
+    dependencies = {
+      "p00f/clangd_extensions.nvim",
```

Comment:
Can you try another structure to make sure clangd_extensions is configurable to the user?

I think it should work :)

Move this as a seperate spec to line 12 for example.

There, add the opts as you specify them below. Also add `lazy=true`.